recipe/migrations/root620000.yaml

```
@@ -3,13 +3,13 @@ __migrator:
   kind:
     version
   migration_number:
-    1
+    2
```

This will force the migration to start over and reissue PRs even for things it has already done. Is that what you want?

---

If the migration never did anything to begin with, you might keep this the same.

---

No, the two which have already been issued are okay.
```suggestion
    1
```

---

I have no idea, I just followed the example from #403. I think @chrisburr has followed the progress of this migration and can maybe comment?

---

Right. Boost was done wrong there IIRC so yeah let's keep it the same.

---

This migration has only discovered one (1) package that needed migrating (which isn't correct anyway), so opening new PRs isn't going to be a big deal.

---

It actually discovered two but one got lost after it was merged: https://github.com/conda-forge/rapidsim-feedstock/pull/2/files